components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3164,9 +3164,10 @@ public static boolean isJITProvisionEnhancedFeatureEnabled() {
      * @throws ClaimManagementException
      */
     public static List<ClaimMapping> getFilteredScopeClaims(List<String> claimListOfScopes,
-                                                            List<ClaimMapping> claimMappings)
+                                                            List<ClaimMapping> claimMappings, String tenantDomain)
             throws ClaimManagementException {
 
+        startTenantFlow(tenantDomain);
```

Comment:
tenant flow needs to be started and ended in a try final blocks

Reply:
Implementation have changes. hence not applicable.